include/common_structs.h

```diff
@@ -863,8 +863,8 @@ typedef struct BattleStatus {
     /* 0x07D */ s8 menuStatus[4]; ///< -1 = automatically pick the first move, 0 = disabled, 1 = enabled
     /* 0x081 */ s8 actionSuccess;
     /* 0x082 */ char unk_82;
-    /* 0x083 */ s8 unk_83;
-    /* 0x084 */ s8 unk_84;
+    /* 0x083 */ s8 actionCommandMode;
```

Comment:
If only used with ActionTutorialModes, call this actionCommandTutorialMode perhaps.

Reply:
After a closer look I figured out that this field is related not only to tutorial, so I changed api function names and enums back to ActionCommandMode